### PR TITLE
[sui-indexer] - getEpochs - add flag to return results in descending order

### PR DESCRIPTION
**Pull request by mamos-mysten**
## Description 

Adds a flag to the `getEpochs` endpoint to allow returning results in descending order

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-indexer/src/lib.rs [hunk 1]
```diff
@@ -72,7 +72,7 @@ pub struct IndexerConfig {
     pub client_metric_port: u16,
     #[clap(long, default_value = "0.0.0.0", global = true)]
     pub rpc_server_url: String,
-    #[clap(long, default_value = "9000", global = true)]
+    #[clap(long, default_value = "3030", global = true)]
```
Comment (patrickkuo) [inline crates/sui-indexer/src/lib.rs:75]: lets keep the default so we don't surprise anyone? don't know if the PE team use the default in our deployment

**Review by patrickkuo — APPROVED** [review]
Nice one, thanks for the fix, just 1 minor comment

File: crates/sui-indexer/src/lib.rs (hunk 1 above)
Comment (mamos-mysten) [reply crates/sui-indexer/src/lib.rs:75]: oh yea - committed this one by mistake!

File: crates/sui-indexer/src/store/pg_indexer_store.rs [hunk 2]
```diff
@@ -1118,16 +1118,28 @@ WHERE e1.epoch = e2.epoch
         &self,
         cursor: Option<EpochId>,
         limit: usize,
+        descending_order: Option<bool>,
     ) -> Result<Vec<EpochInfo>, IndexerError> {
-        let id = cursor.map(|id| id as i64).unwrap_or(-1);
+        let is_descending = descending_order.unwrap_or(false);
```
Comment (gegaowp) [inline crates/sui-indexer/src/store/pg_indexer_store.rs:1123]: nit: unwrap_or_default()

**Review by gegaowp — APPROVED** [review]
looks great, thanks for adding this.

**Pull request merged**